numpy/_core/numerictypes.py

```diff
@@ -359,6 +360,90 @@ def issubsctype(arg1, arg2):
     return issubclass(obj2sctype(arg1), obj2sctype(arg2))
 
 
+@set_module('numpy')
+def isdtype(dtype, kind):
+    """
+    Returns a boolean indicating whether a provided dtype is of a specified
+    data type ``kind``. This function only supports Array API data types.
```

Comment:
This sentence needs updating, no longer true. It should say that all built-in numpy dtypes are supported, and that third-party dtypes are not yet supported.

Reply:
Done!